pytradfri/color.py

```diff
@@ -46,16 +46,16 @@ def supported_features(data: LightResponse) -> int:
     """Return supported features."""
     supported_color_features = 0
 
-    if data.dimmer:
+    if data.dimmer is not None:
         supported_color_features = supported_color_features + SUPPORT_BRIGHTNESS
 
     if data.color_hex:
```

Comment:
Color_hex is a string and thus truthy here, ie we don't need to check for None?

Reply:
I'm assuming that an empty string would be an error/incorrect. But we can change to check for None if we want to allow an empty string.

Reply:
Hmm, I'm not sure here. If I remember correctly, the gateway has a set of permitted hex colors. If one of these colors is set, `color_hex` will contain a hex value. If a color that does not have a string representation is set, then `color_hex` takes some other value (might be "0" as seen here https://github.com/home-assistant-libs/pytradfri/blob/b61737b5241ece653f89f554df7b386d1eb4cb49/tests/devices.py#L31).

I'm inclined to believe that any color bulb would support hex but again, I'm not sure.

Reply:
Ok. I think we can keep it like this for now. If we know better in the future we can update it.